src/shared/cgroup-show.c

```diff
@@ -65,9 +65,9 @@ static void show_pid_array(
         for (i = 0; i < n_pids; i++) {
```

Comment:
(You can switch this to FOREACH_ARRAY BTW)

Reply:
not really, we need the `i` in the body.